install_jupyter_kernel.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+
+legion_dir = os.getcwd() + "/legion"
+sys.path.append(legion_dir)
```

Comment:
Similarly to below, can you use
```
script_dir = os.path.dirname(os.path.realpath(__file__))
```
instead of `os.getcwd()`, so that this script can be run from any directory?

Reply:
And we should probably move this to the `__name__ == "__main__"` block, if possible.